paasta_tools/tron_tools.py

```diff
@@ -235,6 +242,7 @@ def get_spark_config_dict(self):
         aws_creds = get_aws_credentials(
             aws_credentials_yaml=self.config_dict.get("aws_credentials_yaml")
         )
+        paasta_instance = get_smart_paasta_instance_name() or self.get_instance()
```

Comment:
This is unnecessary (meaning you also don't have to define a `get_smart_paasta_instance_name` in tron_tools. Whenever `TronActionConfig` is initialized, we already compose the job and action [here](https://github.com/Yelp/paasta/blob/983ada69507cd1727826ec4f1674e482d21aad89/paasta_tools/tron_tools.py#L571).

Reply:
I've dropped the changes to tron_tools